book/src/puzzle_18/puzzle_18.md

```diff
@@ -28,14 +28,14 @@ Our GPU implementation uses parallel reduction for both finding the maximum valu
 
 ## Configuration
 
-- Vector size: \\(\\text{SIZE} = 128\\)
```

Comment:
Please don't change the convention which we keep for consistency throughout such as using `SIZE`, `TPB` etc.